src/components/GlobalLearningResourcesPage/AppliedFilters.tsx

```diff
@@ -13,13 +13,16 @@ import {
   FiltersMetadata,
 } from '../../utils/FiltersCategoryInterface';
 
+type SetLoaderOptions = (options: FetchQuickstartsOptions) => void;
+type PreviousSetLoaderOptions = (
+  options: (
+    prevLoaderOptions: FetchQuickstartsOptions
+  ) => FetchQuickstartsOptions
+) => void;
+
 const AppliedFilters: React.FC<{
   loaderOptions: FetchQuickstartsOptions;
-  setLoaderOptions: (
-    options: (
-      prevLoaderOptions: FetchQuickstartsOptions
-    ) => FetchQuickstartsOptions
-  ) => void;
+  setLoaderOptions: SetLoaderOptions | PreviousSetLoaderOptions;
```

Comment:
You have the same exact prop defined bellow ith the React generic types. Can you use it here as well?